.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         submodules: recursive
-    - uses: dtolnay/rust-toolchain@1.74.0
+    - uses: dtolnay/rust-toolchain@1.82.0
```

Comment:
1.75 should suffice here and everywhere below (keep the 1.82.0 fixes though).

Problem is that you are forcing anyone who uses the solang or solang-parser crate to update to 1.82.0, when it should work with older versions.

Reply:
Solang depends on a crate which requires `1.75.0`: https://github.com/hyperledger-solang/solang/actions/runs/11792189166/job/32845426551?pr=1664#step:6:505

Should I upgrade to `1.75.0` instead?

Reply:
so it should be the lowest version that works